logging/.env.template

```diff
@@ -0,0 +1,17 @@
+# https://cloud.elastic.co/deployments/814a17235d004d12bb315e8d466e32e3
```

Comment:
the README doesn't address what this is for / how to use it. mind adding a setup section explaining who this is for / what a user needs to modify here?